platform-hub-web/src/app/home/home.module.js

```diff
@@ -2,7 +2,7 @@ import angular from 'angular';
 
 import {HomeComponent} from './home.component';
 
-// Define the app section name
+// Main section component name
 export const AppHome = 'appHome';
 
```

Comment:
Hey @Joseph-Irving – I'm roughly following these guidelines / styleguide: https://github.com/toddmotto/angular-styleguide – it advocates splitting your app up into fairly isolated and decoupled modules (in AngularJS a module is just a grouping of injectables services/factories/components/etc.). The pattern that this particular file follows is: every top level section of the app should have it's own module, and that module is then made a dependency of the main app. That way all specific code for that section goes into it's relevant module, but app-wide shared code goes into the `app.shared` module.